components/List/index.js

```diff
@@ -3,42 +3,42 @@ import Link from "next/link";
 import Image from "next/image";
 
 export default function List({ players }) {
+  function comparePlayers(a, b) {
+    const nameA = a.player.lastname.toLowerCase();
+    const nameB = b.player.lastname.toLowerCase();
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+    return 0;
+  }
+
```

Comment:
I would put it outside of the component, so it doesn't get recreated all the time:

```js
function comparePlayers(a, b) { ... }

export default function List({ players }) { ... }
```

Reply:
Thanks again!